src/core/server/http/types.ts

```diff
@@ -276,8 +285,33 @@ export interface InternalHttpServiceSetup
 
 /** @public */
 export interface HttpServiceStart {
-  /** Indicates if http server is listening on a given port */
-  isListening: (port: number) => boolean;
+  /**
+   * Access or manipulate the Kibana base path
+   * See {@link IBasePath}.
+   */
+  basePath: IBasePath;
+
+  /**
+   * Auth status.
+   * See {@link HttpAuth}
+   */
+  auth: HttpAuth;
+
+  /**
+   * Flag showing whether a server was configured to use TLS connection.
+   */
+  isTlsEnabled: boolean;
```

Comment:
we can ~deprecate~ remove this one in favor of `getServerInfo`

Reply:
Should I add `tls: boolean` to `HttpServerInfo`, or do we assume that `protocol` is good enough?

Reply:
Protocol is enough. Isn't it?

Reply:
AFAIK it is. `protocol==='https'` and `isTlsEnabled===true` are equivalent